api/service/v1alpha1/service.proto

```diff
@@ -61,6 +61,7 @@ service KargoService {
   rpc ApproveFreight(ApproveFreightRequest) returns (ApproveFreightResponse);
   rpc DeleteFreight(DeleteFreightRequest) returns (DeleteFreightResponse);
   rpc GetFreight(GetFreightRequest) returns (GetFreightResponse);
+  rpc WatchFreights(WatchFreightsRequest) returns (stream WatchFreightsResponse);
```

Comment:
This comes up from time to time... Freight is a [mass noun](https://en.wikipedia.org/wiki/Mass_noun).

Grammatically speaking, this should be `WatchFreight` (no "s").

```suggestion
  rpc WatchFreights(WatchFreightsRequest) returns (stream WatchFreightsResponse);
```

Reply:
Implementation looks fine, so LGTM after this is fixed.

Reply:
Almost anywhere you see Freights in code or docs, it is accidental because we've tried to be grammatically correct all along. Wherever we encounter it, I'd like us to fix it, as long as it's not breaking. We don't need to hunt down all such occurrences, rather they can be fixed as we happen upon them while working on other things. There are a few places, like the CLI where "freights" is tolerated due to Postel's law, and there may be a place or two where some lower level package automatically pluralizes a word and doesn't get this one correct. In most cases, we have the ability to be grammatically correct.